example/application/container.tsx

```diff
@@ -3,11 +3,16 @@ import { FrameActions, FrameConnector, HostActionsHandler } from "../../src";
 import { css } from "@emotion/core";
```

Comment:
you should rebase, if not done already, Raymond did changes on this

Reply:
yeah i rebased to latest, b4 making my changes